diff --git a/Install b/Install
index 5cf9ff5..de4a4d7 100755
--- a/Install
+++ b/Install
@@ -20,7 +20,7 @@ cd `dirname $0`
 #
 # Source the configuration file.
 #
-. ./Configuration
+#. ./Configuration
 
 #
 # Verify the arguments to the script.
diff --git a/mgi_cgi.py b/mgi_cgi.py
index 7f9e40a..67cee49 100644
--- a/mgi_cgi.py
+++ b/mgi_cgi.py
@@ -26,7 +26,7 @@ import sys
 import urllib
 import string
 import copy
-import regsub
+import re
 from types import *
 
 
@@ -179,8 +179,7 @@ class FieldStorage:
                         tmpItem = tmpItem[:-1]
                         fields[fieldName]['val'] = [tmpItem]
                     else:
-                        item.value = regsub.gsub(', ', ',',
-                            item.value)
+                        item.value = item.value.gsub(', ', ',')
                         fields[fieldName]['val'] = \
                             string.split(item.value, ',')
                 else: # It's an instance
